internal/model/user_info.go

```diff
@@ -9,6 +9,9 @@ type UserInfo struct {
 	// The users display name.
 	DisplayName string `db:"-" json:"display_name"`
 
+	// The users email address.
+	Emails []string `db:"-" json:"emails"`
```

Comment:
Similar to below. The reason I'm curious is this information is available as soon as 1FA occurs.

Reply:
Yes, should that not be the case? An elevated session is required (by default) to change passwords.

Reply:
Emails are already available after 1FA on all the authz endpoints.